packages/core/src/application/application_ref.ts

```diff
@@ -323,6 +324,7 @@ export class ApplicationRef {
   /** @internal */
   _views: InternalViewRef<unknown>[] = [];
   private readonly internalErrorHandler = inject(INTERNAL_APPLICATION_ERROR_HANDLER);
+  private readonly afterRenderEffectManager = inject(AfterRenderEventManager, {optional: true});
```

Comment:
I believe the `AfterRenderEventManager` would always be present, since it's a tree-shakable provider.
```suggestion
  private readonly afterRenderEffectManager = inject(AfterRenderEventManager);
```